src/slash_commands/mod.rs

```diff
@@ -35,18 +36,19 @@ pub struct SlashCommands {
 }
 
 impl SlashCommands {
-    pub fn new() -> Self {
+    pub fn new(args: &Arguments) -> Self {
         Self {
-            commands: Self::get_slash_commands(),
+            commands: Self::get_slash_commands(args),
         }
     }
 
-    fn get_slash_commands() -> Vec<Box<dyn SlashCommand>> {
-        vec![
-            Box::new(ChomboCommand::new()),
-            Box::new(HandCommand::new()),
-            Box::new(PastaCommand::new()),
-        ]
+    fn get_slash_commands(args: &Arguments) -> Vec<Box<dyn SlashCommand>> {
+        let mut ret: Vec<Box<dyn SlashCommand>> =
+            vec![Box::new(HandCommand::new()), Box::new(PastaCommand::new())];
+        if args.feature_kcc3 {
```

Comment:
Maybe we should allow disabling `/pasta` command as well?

Reply:
heresy